pkg/connector/helper.go

```diff
@@ -105,6 +104,11 @@ func subuserResource(subuser models.Subuser, _ *v2.ResourceId) (*v2.Resource, er
 func teammateInvitationResource(invitation *models.TeammateInvitation) (*v2.Resource, error) {
 	resourceID := strings.TrimSpace(invitation.Token)
 
+	// Validate that the token is not empty after trimming
+	if resourceID == "" {
+		return nil, fmt.Errorf("teammateInvitationResource: empty invitation token")
```

Comment:
should this have an invalid argument grcp code to avoid alerting engineering about the error?